xblock/core.py

```diff
@@ -1,26 +1,33 @@
 """
-Core classes for XBlocks.
+Core classes for the XBlock family.
 
 This code is in the Runtime layer, because it is authored once by edX
 and used by all runtimes.
 
 """
-import functools
 import pkg_resources
-try:
-    import simplesjson as json  # pylint: disable=F0401
-except ImportError:
-    import json
-from webob import Response
-
-from xblock.exceptions import JsonHandlerError, DisallowedFileError
-from xblock.fields import ScopedStorageMixin, HierarchyMixin, String, List, Scope
+
+from xblock.exceptions import DisallowedFileError
+from xblock.fields import String, List, Scope
+from xblock.mixins import ScopedStorageMixin, HierarchyMixin, RuntimeServicesMixin, HandlersMixin
 from xblock.plugin import Plugin
 from xblock.validation import Validation
 
 
 # __all__ controls what classes end up in the docs.
 __all__ = ['XBlock']
+UNSET = object()
```

Comment:
WHY

Reply:
So that any value passed will be recognized, including None.  `object()` gives us a unique object that will fail `x is UNSET` for any possible value passed in as x.   Unless I'm misunderstanding your question?

Reply:
Yeah, I can see that. I don't remember why we were being so cautious.